src/Components/Telemetry.md

```diff
@@ -95,7 +95,7 @@ Aspire.Microsoft.Data.SqlClient:
 - Log categories:
   - none (the client does not provide an easy way to integrate it with logger factory)
 - Activity source names:
-  - "OpenTelemetry.Instrumentation.SqlClient"
+  - "Aspire.Microsoft.Data.SqlClient"
```

Comment:
This should be reverted. https://github.com/dotnet/aspire/pull/3118 will fix it.